packages/harmony/.storybook/docs.tsx

```diff
@@ -1,16 +1,28 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { harmonyDocsThemes } from '../src/storybook/theme'
 import { DocsContainer, DocsContainerProps } from '@storybook/addon-docs'
-import { useDarkMode } from 'storybook-dark-mode'
 import { ThemeProvider } from '../src/foundations/theme'
+import { addons } from '@storybook/preview-api'
+import { UPDATE_DARK_MODE_EVENT_NAME, useDarkMode } from 'storybook-dark-mode'
 
 export const HarmonyDocsContainer = (props: DocsContainerProps) => {
   // @ts-ignore globals are available
   const currentTheme = props.context.store.globals.globals.theme || 'day'
+  const isDark = useDarkMode()
+
+  useEffect(() => {
+    if (isDark && currentTheme === 'day') {
+      addons.getChannel().emit(UPDATE_DARK_MODE_EVENT_NAME)
+    } else if (!isDark && currentTheme !== 'day') {
+      addons.getChannel().emit(UPDATE_DARK_MODE_EVENT_NAME)
```

Comment:
Should both of these be the same?

Reply:
its super awkward but its a "toggle" rather than a set, so this are the conditions we call toggle. could have been a single if statement, but the condition was getting rather long.